wallet/src/actors/worker/methods.rs

```diff
@@ -19,26 +18,11 @@ impl Worker {
         })
     }
 
-    pub fn run_rad_request(&self, request: types::RADRequest) -> Result<types::RadonTypes> {
-        // Block on data request retrieval because the wallet was designed with a blocking
-        // run_retrieval in mind.
-        // This can be made non-blocking by returning a future here and updating
```

Comment:
You can leave this part of comment (last 2 lines) as it is still relevant

Reply:
Looks to me like the the latest changes doesn't really match the suggestion :sweat_smile: